vite.config.js

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
+import pkg from './package.json'
+
+export default defineConfig({
+  plugins: [vue()],
+  build: {
+    target: 'esnext',
+    minify: false,
+    rollupOptions: {
+      external: ['vue']
```

Comment:
For some reason adding all of `pkg.dependencies` here breaks treeshaking in consumer apps 😕